docs/product-installation-guide/src/main/asciidoc/eap-execution-server-configure-proc.adoc

```diff
@@ -1,36 +1,68 @@
 [id='eap_execution_server_configure_proc']
 = Configuring {KIE_SERVER} 
 
-If {KIE_SERVER} will be managed by {CENTRAL}, you must edit the `standalone.xml` file as described in this section.
+If {KIE_SERVER} will be managed by {CENTRAL}, you must edit the `standalone.xml` file in both the {KIE_SERVER} and {CENTRAL} installations, as described in this section.
 
 [NOTE]
 ====
-Only make these changes if {KIE_SERVER} will be managed by {CENTRAL}.
+Only make these changes if {KIE_SERVER} will be managed by {CENTRAL}. 
 ====
 
-Prerequisites
-{PRODUCT} installed in an  `__EAP_HOME__` as described in one of the following sections:
-
-* <<installer-run-proc>>
-* <<installer-run-cli-proc>>
-* <<assembly_installing-on-eap-deployable>>
+.Prerequisites
```

Comment:
Line break issue in output (Prerequisites Red Hat ...). Should be singular Prerequisite and the first line should not be bulleted. That might fix it anyway.